### DISCUSSION TITLE
Quiet execution of the install bash file

### DISCUSSION THREAD

**Discussion by lamyc** [ayamir/nvimdots#580] [Q&A] [answered]
Hi, thanks for your great work on nvimdots. It made me appreciate how amazing nvim can be.

I would like to ask how or if it is possible to accept all the defaults and install silently.
Currently, I am trying to write ansible playbooks for my dotfiles. I am asking this question so that I can have it automatically installed.

**Comment by Jint-lzxy** [comment, marked as answer]
Yep that's possible, and we have already utilized this feature in our [CI pipeline](https://github.com/ayamir/nvimdots/blob/b05e62ed931011f99cf2e5e5ef4b0d4ad8a59415/.github/workflows/update_lockfile.yml#L27-L33). You only need to "pretend" that the current session is non-interactive _(i.e., define `${NONINTERACTIVE}` OR `${CI}`)_ during installation:
https://github.com/ayamir/nvimdots/blob/b05e62ed931011f99cf2e5e5ef4b0d4ad8a59415/scripts/install.sh#L174-L192
Note that in non-interactive mode, the script will default-select the following options:
- `use_ssh = false` (use HTTPS);
- Enforce all commands;
- Answer "Yes" to all prompts.
